File: app/src/main/java/com/example/android/miniweather/Network/WeatherApi.java
package com.example.android.miniweather.Network;

import com.example.android.miniweather.Models.CityWeather;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface WeatherApi {

    @GET("v1/forecast.json")
    Call<CityWeather> getWeather(@Query("key") String apiKey,
                                 @Query("q") String city,
                                 @Query("days") int days);
}

File: app/src/main/java/com/example/android/miniweather/network/WeatherApi.kt
package com.example.android.miniweather.network

import com.example.android.miniweather.model.CityWeather

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface WeatherApi {

    @GET("v1/forecast.json")
    fun getWeather(@Query("key") apiKey: String,
                   @Query("q") city: String,
                   @Query("days") days: Int): Call<CityWeather>
}
